packages/api/src/lcd.ts

```diff
@@ -0,0 +1,30 @@
+import Service from './service-lcd'
+import Instance from './instance-lcd'
+import Runner from './runner-lcd'
+import Process from './process-lcd'
+import Execution from './execution-lcd'
+import Ownership from './ownership-lcd'
+import Account from './account-lcd'
+
+class API {
+  service: Service
+  instance: Instance
+  runner: Runner
+  process: Process
+  execution: Execution
+  ownership: Ownership
+  account: Account
+
+  constructor(endpoint: string = "http://localhost:1317") {
```

Comment:
is it really necessary?
```suggestion
  constructor(endpoint: string) {
```